shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.js

```diff
@@ -4,40 +4,73 @@ import * as ReactDOM from 'react-dom'
 import {resolveRootAsURL} from '../../../../../desktop/app/resolve-root.desktop'
 import {urlsToImgSet} from '../../../../../common-adapters/icon.desktop'
 import {Box, ConnectedUsernames, Text} from '../../../../../common-adapters'
-import {collapseStyles, globalColors, styleSheetCreate} from '../../../../../styles'
+import {
+  collapseStyles,
+  glamorous,
+  globalColors,
+  platformStyles,
+  styleSheetCreate,
+} from '../../../../../styles'
```

Comment:
so whats the native story here?

Reply:
Nothing on native yet, I'll look into it today